pkg/payment/request.go

```diff
@@ -247,6 +250,18 @@ type RulesRequest struct {
 	Discounts []DiscountRequest `json:"discounts,omitempty"`
 }
 
+// AuthenticationRequest represents payment authentication request within DataRequest.
+type AuthenticationRequest struct {
+	Type                 string `json:"type,omitempty"`
+	Cryptogram           string `json:"cryptogram,omitempty"`
+	ThreeDSServerTransID string `json:"three_ds_server_trans_id,omitempty"`
+	ECI                  string `json:"eci,omitempty"`
+	DSTransID            string `json:"ds_trans_id"`
```

Comment:
por que esse nao tem o omit empty?
```suggestion
	DSTransID            string `json:"ds_trans_id,omitempty"`
```

Reply:
Era pra ter. Corrigido.